x-pack/legacy/plugins/siem/public/components/embeddables/embeddable.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiPanel } from '@elastic/eui';
+import React from 'react';
+import styled from 'styled-components';
+
+const Panel = styled(EuiPanel).attrs({
+  paddingSize: 'none',
+})`
+  overflow: hidden;
+`;
```

Comment:
Agh! I know `styled-components` are on their outs, but should we be using `attrs` like you are here?

Just looked [in the docs](https://www.styled-components.com/docs/faqs#when-to-use-attrs), and seems so:

> The rule of thumb is to use attrs when you want every instance of a styled component to have that prop, and pass props directly when every instance needs a different one.

That's nice too as you get linting since it's not in a template!

Reply:
I can change to inline for the `paddingSize` prop.